src/PostgREST/CLI.hs

```diff
@@ -46,8 +46,21 @@ main CLI{cliCommand, cliPath} = do
       CmdDumpConfig -> do
         when configDbConfig $ AppState.reReadConfig True appState (const $ pure ())
         putStr . Config.toText =<< AppState.getConfig appState
-      CmdDumpSchema -> putStrLn =<< dumpSchema appState
+      CmdDumpSchemaCache -> putStrLn =<< dumpSchema appState
       CmdRun -> App.run appState (Logger.logObservation loggerState))
+main CLIExample{exampleType} = do
+  conf <-
+    either panic identity <$> Config.readAppConfig mempty mempty Nothing mempty mempty
+
+  loggerState <- Logger.init
+  bracket
+    (AppState.init conf $ Logger.logObservation loggerState)
+    AppState.destroy
+    (\_ -> case exampleType of
+      "file" -> putStrLn exampleConfigFile
+      "db" -> putStrLn exampleConfigDb
+      "env" -> putStrLn exampleConfigEnv
+      _     -> putStrLn ("Usage: postgrest-run --example/-e file/db/env" :: [Char]))
```

Comment:
I don't think you need to duplicate this code. You could keep `main CLI{...}` and maybe add another `cliCommand` like `CMDExample`, then the code to print the examples could be done in a `printExample` function (like `dumpSchema`, for instance).